src/DynamoCore/DynamoCore.csproj

```diff
@@ -86,6 +86,7 @@ limitations under the License.
       <HintPath>..\packages\Unofficial.Ionic.Zip.1.9.1.8\lib\Ionic.Zip.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Microsoft.CSharp" />
```

Comment:
any memory of why this is required?

Reply:
Needed when 'dynamic' is used.

Reply:
@sm6srw hmm, can you verify this does not break the mono build?

Reply:
I think the Mono build is fine but I will check again.